### PR TITLE
{ARM} Bump `MGMT_RESOURCE_SUBSCRIPTIONS` to 2022-12-01

### PR DESCRIPTION
**Pull request by jiasli**
**Related command**
`az login`

**Description**
Bump `MGMT_RESOURCE_SUBSCRIPTIONS` to 2022-12-01 in order to consume `defaultDomain` and `displayName` properties from [Tenants - List](https://learn.microsoft.com/en-us/rest/api/resources/tenants/list?view=rest-resources-2022-12-01&tabs=HTTP) API.

### CODE REVIEW

**Comment by yonzhan** [comment]
ARM

**Comment by jiasli** [comment]
`HDInsightClusterTests::test_hdinsight_application` fails with

```
        if response.status_code not in [200]:
            map_error(status_code=response.status_code, response=response, error_map=error_map)
            error = self._deserialize.failsafe_deserialize(_models.ErrorResponse, response)
>           raise HttpResponseError(response=response, model=error, error_format=ARMErrorFormat)
E           azure.core.exceptions.HttpResponseError: (BadRequest) User SubscriptionId '0b1f6471-1bf0-4dda-aec3-cb9272f09590' does not have cores left to create resource 'hdicli-etapsnumb'. Required: 40, Available: 24.
E           Code: BadRequest
E           Message: User SubscriptionId '0b1f6471-1bf0-4dda-aec3-cb9272f09590' does not have cores left to create resource 'hdicli-etapsnumb'. Required: 40, Available: 24.

../py310/lib/python3.10/site-packages/azure/mgmt/hdinsight/operations/_clusters_operations.py:95: HttpResponseError
```

File: src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_image_builder_commands.py [hunk 1]
```diff
@@ -658,6 +658,7 @@ def test_image_build_trigger(self, resource_group):
                  ])
         self.cmd('image builder trigger delete --image-template-name {tmpl} -g {rg} --trigger-name {trigger} --yes')
 
+    @unittest.skip('https://github.com/Azure/azure-cli/issues/28677')
```
Comment (jiasli) [inline src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_image_builder_commands.py:661]: https://github.com/Azure/azure-cli/issues/28677

File: src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py [hunk 2]
```diff
@@ -2663,6 +2663,7 @@ def test_vmss_extension_instance_name(self):
 
 class VMSSExtensionImageTest(ScenarioTest):
 
+    @live_only()  # https://github.com/Azure/azure-cli/issues/28678
```
Comment (jiasli) [inline src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py:2666]: https://github.com/Azure/azure-cli/issues/28678

File: src/azure-cli/azure/cli/command_modules/hdinsight/tests/latest/test_hdinsight_commands.py [hunk 3]
```diff
@@ -50,6 +50,7 @@ def test_hdinsight_cluster_kafka(self, storage_account_info):
 
     # Uses 'rg' kwarg
     # _rest_proxy_arguments() will override location to southcentralus, so use this location for rg and sa
+    @unittest.skip('https://github.com/Azure/azure-cli/issues/28860')
```
Comment (jiasli) [inline src/azure-cli/azure/cli/command_modules/hdinsight/tests/latest/test_hdinsight_commands.py:53]: https://github.com/Azure/azure-cli/issues/28860

**Comment by jiasli** [comment]
#28803 caused merge conflict on `src/azure-cli/azure/cli/command_modules/containerapp/tests/latest/recordings/test_containerapp_custom_domains_app_in_different_rg.yaml`.

**Comment by jiasli** [comment]
https://github.com/Azure/azure-cli/pull/28899 causes merge conflicts on

```
src/azure-cli/azure/cli/command_modules/appservice/tests/latest/recordings/test_functionapp_vnetE2E.yaml
src/azure-cli/azure/cli/command_modules/appservice/tests/latest/recordings/test_functionapp_vnet_EP_sku_E2E.yaml
src/azure-cli/azure/cli/command_modules/appservice/tests/latest/recordings/test_functionapp_vnet_basic_sku_E2E.yaml
src/azure-cli/azure/cli/command_modules/appservice/tests/latest/recordings/test_functionapp_vnet_duplicate_name.yaml
src/azure-cli/azure/cli/command_modules/appservice/tests/latest/recordings/test_webapp_vnetDelegation.yaml
src/azure-cli/azure/cli/command_modules/appservice/tests/latest/recordings/test_webapp_vnetE2E.yaml
src/azure-cli/azure/cli/command_modules/appservice/tests/latest/recordings/test_webapp_vnetRouteAll.yaml
src/azure-cli/azure/cli/command_modules/appservice/tests/latest/recordings/test_webapp_vnetSameName.yaml
src/azure-cli/azure/cli/command_modules/appservice/tests/latest/recordings/test_webapp_vnetSubnetId.yaml
```

**Comment by jiasli** [comment]
https://github.com/Azure/azure-cli/pull/28847 causes merge conflicts on

```
src/azure-cli/azure/cli/command_modules/vm/tests/latest/recordings/test_vm_create_existing_ids_options.yaml
src/azure-cli/azure/cli/command_modules/vm/tests/latest/recordings/test_vm_create_existing_options.yaml
src/azure-cli/azure/cli/command_modules/vm/tests/latest/recordings/test_vm_image_list_thru_services_edge_zone_by_arch.yaml
src/azure-cli/azure/cli/command_modules/vm/tests/latest/recordings/test_vm_image_list_thru_services_filtered_by_arch.yaml
src/azure-cli/azure/cli/command_modules/vm/tests/latest/recordings/test_vm_image_terms.yaml
src/azure-cli/azure/cli/command_modules/vm/tests/latest/recordings/test_vm_images_list_thru_services_edge_zone.yaml
src/azure-cli/azure/cli/command_modules/vm/tests/latest/recordings/test_vmss_create_existing_ids_options.yaml
src/azure-cli/azure/cli/command_modules/vm/tests/latest/recordings/test_vmss_create_existing_options.yaml
```

File: src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_image_builder_commands.py [hunk 4]
```diff
@@ -28,6 +28,7 @@
 INDEX_FILE = "https://raw.githubusercontent.com/danielsollondon/azvmimagebuilder/master/quickquickstarts/exampleArtifacts/buildArtifacts/index.html"
 
 
+@unittest.skip("https://github.com/Azure/azure-cli/issues/28909")
```
Comment (jiasli) [inline src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_image_builder_commands.py:31]: https://github.com/Azure/azure-cli/issues/28909

File: src/azure-cli/azure/cli/command_modules/containerapp/tests/latest/test_containerapp_commands.py [hunk 5]
```diff
@@ -238,6 +238,7 @@ def test_containerapp_ingress_traffic_e2e(self, resource_group):
         for revision in revisions_list:
             self.assertEqual(revision["properties"]["trafficWeight"], 50)
 
+    @unittest.skip('https://github.com/Azure/azure-cli/issues/28680')
```
Comment (jiasli) [inline src/azure-cli/azure/cli/command_modules/containerapp/tests/latest/test_containerapp_commands.py:241]: - https://github.com/Azure/azure-cli/issues/28680

**Review by evelyn-ys — APPROVED** [review]

**Review by bebound — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]

**Review by jsntcy — APPROVED** [review]

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**